svthalia/concrexit
# Promotion publish reminder

Closes #3340



### Summary
Added the promotion daily overview that it groups the emails by receiver.

### How to test
Steps to test the changes you made:
1. Ensure there is a promotion request scheduled to be published today
2. poetry run python website/manage.py sendpromodailyoverview
3. Ensure that the correct mail is sent (check in terminal output)
